packages/api-contract/src/Abi/toLatest.spec.ts

```diff
@@ -97,4 +107,12 @@ describe('v3ToLatest', (): void => {
       latest.spec.constructors[1].payable.isTrue
     ).toEqual(true);
   });
+
+  it('has the correct messages', (): void => {
+    const contract = registry.createType('ContractMetadata', { V3: abis.ink_v3_trait_erc20.V3 });
```

Comment:
```suggestion
    const contract = registry.createType('ContractMetadata', { V3: abis.ink_v3_traitErc20.V3 });
```

Update for lint fix #3